api/internal/target/kusttarget.go

```diff
@@ -338,18 +341,27 @@ func (kt *KustTarget) configureExternalTransformers(transformers []string) ([]re
 			// not an inline config
 			transformerPaths = append(transformerPaths, p)
 			continue
+		} else {
+			// inline config, track the origin
```

Comment:
nit: the `else` condition seems not necessary

Reply:
done